src/app/shared/substitutor/label-substitutor.directive.spec.ts

```diff
@@ -420,8 +420,26 @@ describe('LabelSubstitutorDirective', () => {
             });
             fixture.detectChanges();
 
-            expect(labelSubstitutionService.substituteLabel).toHaveBeenCalledWith({ LabelB: '', LabelA: '07-03-2018' }, label);
+            expect(labelSubstitutionService.substituteLabel).toHaveBeenCalledWith({ LabelB: '', LabelA: '7 Mar 2018' }, label);
         });
+
+        it('should pass form field value with invalid date', () => {
```

Comment:
Should we be consistent with the test naming convention above?

Reply:
Done